distribution/options.html

```diff
@@ -64,6 +64,20 @@
 				<button id="clear-cache">Clear cache</button>
 			</label>
 		</p>
+		<p>
+			<strong>Find a feature</strong>
+		</p>
+		<p>
+			This process will help you identify what Refined GitHub feature is making changes or causing issues on GitHub.
+		</p>
+		<p>
+			<label>
+				<button id="find-feature">Find feature…</button>
+			</label>
+		</p>
+		<p id="find-feature-message" hidden>
```

Comment:
I think this should actually go above the button, so when clicking it, the button jumps down. This also helps avoid an issue where the button lines up with the bottom border, so the new message wouldn't be visible.